image/v1/types.go

```diff
@@ -670,6 +710,8 @@ type ImageImportStatus struct {
 	Image *Image `json:"image,omitempty" protobuf:"bytes,2,opt,name=image"`
 	// Tag is the tag this image was located under, if any
 	Tag string `json:"tag,omitempty" protobuf:"bytes,3,opt,name=tag"`
+	// Manifests holds sub-manifests metadata when importing a manifest list
+	Manifests []Image `json:"manifests,omitempty" protobuf:"bytes,4,rep,name=manifests"`
```

Comment:
Why does a Manifests field not use the new ImageManifest type?

Reply:
the `ImageManifest` type is a representation of a sub-manifest in a manifest list. It contains less information about the manifest than what is represented by the `Image` type - it's basically a glorified pointer identifying a manifest by platform.

That's also why we use the `Image` type here, because we want the whole manifest (including config, layers, etc).

These links may help:
* Image index (manifest list) spec. Special note to the `manifests` list: https://github.com/opencontainers/image-spec/blob/main/image-index.md#image-index-property-descriptions
* manifest spec, note it contains a lot more information than then stuff in the `manifests` list items in the image index: https://github.com/opencontainers/image-spec/blob/main/manifest.md#image-manifest-property-descriptions